opscode-cookbooks/firewall
# Update testing and add more tests

- Add missing logging method in provider/rule_ufw
- Move unit tests into test/unit (update Rakefile accordingly)
- Update from deprecated chefspec matcher definition syntax
- Add TK tests that use serverspec to verify ufw rules are in place
- Add actual converge for chefspec and a test to be sure ufw package is installed
- Rename 'fake' cookbook to firewall-test
- Update Berksfile with newer API endpoint URLs
